Client/Frontend/Browser/BrowserScrollController.swift

```diff
@@ -37,12 +37,14 @@ class BrowserScrollingController: NSObject {
     weak var snackBars: UIView?
 
     var footerBottomConstraint: Constraint?
-    var headerTopConstraint: Constraint?
+    // TODO: Since SnapKit hasn't added support yet (Swift 2.0/iOS 9) for handling layoutGuides,
+    // this constraint uses the system abstraction instead of SnapKit's Constraint class
+    var headerTopConstraint: NSLayoutConstraint?
```

Comment:
Found [this](https://github.com/SnapKit/SnapKit/commit/6252444966517bd22c9bb673c8a41d8d1c7306ea). Does that mean we could use `snp_topLayoutGuideTop`?